etc/docker/dev/docker-compose-storage.yml

```diff
@@ -0,0 +1,81 @@
+version: "2"
+services:
+  rucio:
+    image: rucio/rucio-dev
+    hostname: rucio
+    ports:
+      - "443:443"
+    links:
+      - ruciodb:ruciodb
+      - graphite:graphite
+      - fts:fts
+      - ftsdb:ftsdb
+      - xrd1:xrd1
+      - xrd2:xrd2
+      - xrd3:xrd3
+    volumes:
+      - ../../../tools:/opt/rucio/tools
+      - ../../../bin:/opt/rucio/bin
+      - ../../../lib:/opt/rucio/lib
+    environment:
+      - X509_USER_CERT=/opt/rucio/etc/usercert.pem
+      - X509_USER_KEY=/opt/rucio/etc/userkey.pem
+  ruciodb:
+    image: postgres:11
+    environment:
+      - POSTGRES_USER=rucio
+      - POSTGRES_DB=rucio
+      - POSTGRES_PASSWORD=secret
+    ports:
+      - "5432:5432"
+    command: ["-c", "fsync=off","-c", "synchronous_commit=off","-c", "full_page_writes=off"]
+  graphite:
+    image: graphiteapp/graphite-statsd
+    ports:
+      - "80:80"
+  fts:
+    image: rucio/fts
```

Comment:
Maybe using the docker images provided by FTS instead/ see https://gitlab.cern.ch/fts/fts3/tree/develop/packaging/docker
```
image: gitlab-registry.cern.ch/fts/fts3:latest
image: gitlab-registry.cern.ch/fts/fts-rest:latest
gitlab-registry.cern.ch/fts/fts-monitoring:latest
```

Reply:
yeah, we cooked up our own because the gitlab registry at CERN was having some problems with outside connections. i think that's much better now, so most likely i'll migrate that soon